clients/fides-js/src/components/fides.css

```diff
@@ -1023,13 +1023,13 @@ div#fides-overlay-wrapper .fides-toggle .fides-toggle-display {
   position: relative;
 }
 
-.fides-modal-container .fides-i18n-menu {
+.fides-modal-footer .fides-i18n-menu {
```

Comment:
![Screenshot 2024-12-06 at 13 52 34](https://github.com/user-attachments/assets/74e8f885-eac4-4bbe-868c-d9ecc2d5ecf1)

These styles still don't seem to be applying for me and the i18n menu is still centered, is that intentional?

Reply:
weird, that was working for me. Try rebuilding/clearing your cache?